index.html

```diff
@@ -25,6 +26,15 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav">
+				<ul>
+					<li><a class="nav-link" href="#projects">About</a></li>
+					<li><a class="nav-link" href="#about-me">About</a></li>
+					<li>
+						<a class="nav-link" href="#contact">Contact</a>
+					</li>
+				</ul>
+			</nav>
```

Comment:
- [ ] You did a great job here wrapping your menu links in the `<ul>` list. The menu is appearing by clicking on the hamburger icon But to make this menu fully functional there should be a button that can use to close the mobile menu. Please add a button to this nav container.